src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFragmentStateAdapter.cs

```diff
@@ -57,6 +84,7 @@ protected override void Dispose(bool disposing)
 			{
 				SectionController.ItemsCollectionChanged -= OnItemsCollectionChanged;
 				_shellSection = null;
+				_items = null;
```

Comment:
maybe we should also do this for the dictionary?

Reply:
or at least clear both lists.

Reply:
the `_items` is a readonly list that comes from the xplat code so I can't clear it. But I'll clear the other one